classes/php.php

```diff
@@ -41,6 +41,11 @@ public function setBinaryPath($phpPath = null)
 			}
 		}
 
+		if (basename($phpPath) === 'hhvm')
```

Comment:
I don't have a project with an atoum test suite. I was considering it, but then decided against it because hhvm didn't work at all.

I suggest you make the travis build work (at least for native php) and then merge so people start running their suites on hhvm. If there are still issues nothing is getting worse for them, considering that hhvm doesn't even start currently. But they may report (and possibly even fix!) any remaining problems.